apps/aecontract/test/aect_test_utils.erl

```diff
@@ -414,12 +415,13 @@ legacy_compile(Vsn, SrcFile) ->
 compiler_cmd(Vsn) ->
     BaseDir = filename:join([code:priv_dir(aesophia_cli), "bin"]),
     case Vsn of
-        ?SOPHIA_ROMA      -> filename:join([BaseDir, "v1.4.0", "aesophia_cli"]);
-        ?SOPHIA_MINERVA   -> filename:join([BaseDir, "v2.1.0", "aesophia_cli"]);
-        ?SOPHIA_FORTUNA   -> filename:join([BaseDir, "v3.2.0", "aesophia_cli"]);
-        ?SOPHIA_LIMA_AEVM -> filename:join([BaseDir, "v4.3.1", "aesophia_cli"]) ++ " --backend=aevm";
-        ?SOPHIA_LIMA_FATE -> filename:join([BaseDir, "v4.3.1", "aesophia_cli"]);
-        ?SOPHIA_IRIS_FATE -> filename:join([BaseDir, "v6.1.0", "aesophia_cli"])
+        ?SOPHIA_ROMA       -> filename:join([BaseDir, "v1.4.0", "aesophia_cli"]);
+        ?SOPHIA_MINERVA    -> filename:join([BaseDir, "v2.1.0", "aesophia_cli"]);
+        ?SOPHIA_FORTUNA    -> filename:join([BaseDir, "v3.2.0", "aesophia_cli"]);
+        ?SOPHIA_LIMA_AEVM  -> filename:join([BaseDir, "v4.3.1", "aesophia_cli"]) ++ " --backend=aevm";
+        ?SOPHIA_LIMA_FATE  -> filename:join([BaseDir, "v4.3.1", "aesophia_cli"]);
+        ?SOPHIA_IRIS_FATE  -> filename:join([BaseDir, "v6.1.0", "aesophia_cli"]);
+        ?SOPHIA_CERES_FATE -> filename:join([BaseDir, "v6.1.0", "aesophia_cli"])
```

Comment:
Not v7.0.0?

Reply:
There is no v7.0.0 of the CLI compiler yet, and Ceres shouldn't need the CLI compiler either - my guess is that there is a poorly written test case somewhere that does the wrong thing... I'll check.

Reply:
The caching/aci seems to use this under some circumstances... I won't try to untangle that mess now...